docs/buildtest_perlmutter.rst

```diff
@@ -0,0 +1,116 @@
+Buildtest Tutorial on Perlmutter
+===================================
+
+This tutorial will be conducted on `Perlmutter <https://docs.nersc.gov/systems/perlmutter/>`_ system. If you need account access please
+`obtain a user account <https://docs.nersc.gov/accounts/>`_.
+
+Setup
+------
+
+Once you have a NERSC account, you can `connect to NERSC system <https://docs.nersc.gov/connect/>`_. You will need access to a
+terminal client and ssh into perlmutter as follows::
+
+    ssh <user>@perlmutter-p1.nersc.gov
+
+To get started please load the **python** module since you will need python 3.7 or higher to use buildtest. This can be done by running::
+
+    module load python
+
+Next, you should :ref:`Install buildtest <installing_buildtest>` by cloning the repository in your $HOME directory.
+
+Once you have buildtest setup, please clone the following repository https://github.com/buildtesters/buildtest-nersc in your $HOME directory as follows::
+
+    cd $HOME
+    git clone https://github.com/buildtesters/buildtest-nersc $HOME/buildtest-nersc
+    export BUILDTEST_CONFIGFILE=$HOME/buildtest-nersc/config.yml
+
+Once you are done, please navigate back to root of buildtest which can be done by running::
+
+    cd $BUILDTEST_ROOT
+
+Exercise 1: Running a Batch Job
+--------------------------------
+
+In this exercise, we will submit a batch job that will run `hostname` in the slurm cluster. Shown below is the example buildspec
+
+.. literalinclude:: ../perlmutter_tutorial/ex1/hostname.yml
+   :language: yaml
+
+Let's run this test and poll interval for 10 secs::
+
+   buildtest build -b $BUILDTEST_ROOT/perlmutter_tutorial/ex1/hostname.yml --pollinterval=10
+
+Once test is complete, check the output of test by running::
```

Comment:
Suggestion: "Once *the* test is complete, *you* can check the output of *the* test by running"